Sources/ShortcutUI/Extensions/Button+Extension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Button+Extension.swift
+//  ShortcutUI
+//
+//  Created by Sina Rabiei on 2022-05-23.
+//  Copyright © 2022 Shortcut Scandinavia Apps AB. All rights reserved.
+
+import SwiftUI
+
+public extension Button {
+    init(priority: TaskPriority? = nil, action: @escaping () async -> Void, label: () -> Label) {
```

Comment:
Should we add `@ViewBuilder` just before `label` in this init, just to make it perfectly clear? That would make it similar to the normal `init` for `Button`:
```
    public init(action: @escaping () -> Void, @ViewBuilder label: () -> Label)
```
This is just an idea from my part, I don't know if this is really necessary (but I guess it couldn't hurt).

Reply:
Good point. I'll add this.